docs/remixd.md

```diff
@@ -1,68 +1,92 @@
 Remixd: Access your Local Filesystem 
 =========================================
-To give the Remix-ide (the web app) access to a folder on your local computer, you need to use `remixd`.  
+To give the Remix IDE (the web app) access to a folder on your computer, you need to use both **Remixd** and **remixd**.  
```

Comment:
Why is remixd menitoned twice?

Reply:
We can say 'you need to use **Remixd** plugin alongside **remixd** cli/npm module.'